src/components/UploadForm.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+const UploadForm=()=>{
+    const[selectedFile, setSelectedFile] = ("");
```

Comment:
Is this supposed to be `const [selectedFile, setSelectedFile] = React.useState("");`?

Reply:
whoops you are correct